cockroachcloud/export-logs.md

```diff
@@ -262,13 +262,15 @@ Perform the following steps to enable log export from your {{ site.data.products
     - `{your_cluster_id}` is the cluster ID of your {{ site.data.products.dedicated }} cluster as determined in step 2.
     - `{secret_key}` is your API access key. See [API Access](console-access-management.html) for more details.
 
-    The resulting log export service account should resemble the following example:
+    The resulting GCP principal should resemble the following example:
 
     {% include_cached copy-clipboard.html %}
     ~~~
     crl-logging-user-a1c42be2e53b@crl-prod-abc.iam.gserviceaccount.com
     ~~~
 
+    This GCP principal refers to a resource that is owned by Cockroach Labs and is created automatically along with your cluster. You **do not** need to create this account in GCP; it is already present for use by your cluster.
+
 1. Create a new role with the required permissions in your GCP project:
```

Comment:
Do you think it's worth splitting the directions more severely between "stuff you do with CRL API" and "stuff you do with GCP". These steps mix both and I wonder if that can be clarified